File: ALODWebUtility/Common/MemberSelectedEventArgs.cs
using System;

namespace ALODWebUtility.Common
{
    public class MemberSelectedEventArgs : EventArgs
    {
        private int _selectedRowIndex;

        public int SelectedRowIndex
        {
            get { return _selectedRowIndex; }
            set { _selectedRowIndex = value; }
        }
    }
}

File: ALODWebUtility/Common/MemberSelectedEventArgs.vb
Namespace Common

    Public Class MemberSelectedEventArgs
        Inherits System.EventArgs

        Private _selectedRowIndex As Integer

        Public Property SelectedRowIndex() As Integer
            Get
                Return _selectedRowIndex
            End Get
            Set(value As Integer)
                _selectedRowIndex = value
            End Set
        End Property

    End Class

End Namespace